### DISCUSSION TITLE
Hardware and System recommendation

### DISCUSSION THREAD

**Discussion by Mannshoch** [games-on-whales/wolf#141] [Q&A]
I have the Idea to create a Game Server for my ancient Windows Games from Win 95 until 8. I tested it on my mint (deb installation) with Lutris and Sunshine. It works greate. 

I need hardware (link to product page if possible) and system recommendateions. 

Assumed things to consider:

- Fast wake from WOL
- Good support for Directx over Vulkan
- Freecad, 0AD, Gnome Shotwell, Tux race are shared Linux software
- Three parallel streams from three different user possible (with their own game settings/home? With some exception)
- Passive GPU (If resonable) 
- No Screen attached solution

**Comment by ConsularParadi** [comment]
Hey I am not sure if this is the right platform to ask for hardware recommendations. You can hop on reddit and search for communities such as cloudygamer, selfhost, homelab. They might have a better idea about hardware.
